src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -201,15 +197,31 @@ const ChatItemMyTextV2 = ({
   }, []);
 
   const renderAvatar = React.useCallback(() => {
-    if (isContinuous) return <View style={styles.avatar} />;
+    if (isContinuous) return <View style={[styles.avatar, styles.ml8]} />;
     return <View style={styles.ml8}>{avatar}</View>;
   }, []);
 
   const handleTextContainerStyle = () => {
     if (chatType === SIGNED) {
-      return [styles.containerSigned, styles.textContainer];
+      return [
+        styles.containerSigned,
+        styles.textContainer,
+        isContinuous
+          ? {}
+          : {
+              borderTopEndRadius: 0
+            }
+      ];
     }
-    return [styles.containerAnon, styles.textContainer];
+    return [
+      styles.containerAnon,
+      styles.textContainer,
+      isContinuous
+        ? {}
+        : {
+            borderTopEndRadius: 0
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `renderAvatar` function, the `ml8` style is now included when `isContinuous` is true. This will add additional margin to the avatar in continuous messages. Make sure this change doesn't disrupt the layout.
In the `handleTextContainerStyle` function, the borderTopEndRadius is now set to 0 when `isContinuous` is false. This could potentially affect the appearance of chat bubbles. Ensure that this change aligns with your design guidelines.
```diff
- if (isContinuous) return <View style={styles.avatar} />;
+ if (isContinuous) return <View style={[styles.avatar, styles.ml8]} />;

- return [styles.containerSigned, styles.textContainer];
+ return [
+   styles.containerSigned,
+   styles.textContainer,
+   isContinuous
+     ? {}
+     : {
+         borderTopEndRadius: 0
+       }
+ ];
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->